app/src/main/res/layout/screen_patient_summary.xml

```diff
@@ -146,6 +146,15 @@
         android:layout_marginEnd="@dimen/spacing_8"
         android:visibility="gone" />
 
+      <org.simple.clinic.summary.assignedfacility.AssignedFacilityView
+        android:id="@+id/assignedFacilityView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/spacing_8"
+        android:layout_marginTop="@dimen/spacing_8"
+        android:layout_marginEnd="@dimen/spacing_8"
+        android:visibility="gone" />
```

Comment:
We are setting the visibility of this in the parent layout but toggling the visibility in the child view. We should be handling this in the same layer.

Reply:
Yeah, moving this to `PatientSummaryScreen`. I will raise a separate PR for this since this PR already has several changes.

Reply:
NVM, it's a simple change. I will do it in this PR